cmd/minishift/cmd/util/util.go

```diff
@@ -35,17 +36,17 @@ func VMExists(client *libmachine.Client, machineName string) bool {
 func ExitIfUndefined(client *libmachine.Client, machineName string) {
 	exists := VMExists(client, machineName)
 	if !exists {
-		atexit.ExitWithMessage(0, fmt.Sprintf("There is currently no '%s' VM defined.", machineName))
+		atexit.ExitWithMessage(0, fmt.Sprintf("The execution of this command requires an existing '%s' VM. However, there is currently none defined.", machineName))
 	}
 }
 
 func IsHostRunning(driver drivers.Driver) bool {
 	return drivers.MachineInState(driver, state.Running)()
 }
 
-func ExitIfNotRunning(driver drivers.Driver) {
+func ExitIfNotRunning(driver drivers.Driver, machineName string) {
 	running := IsHostRunning(driver)
 	if !running {
-		atexit.ExitWithMessage(0, "Minishift VM is not running.")
+		atexit.ExitWithMessage(0, fmt.Sprintf("The execution of this command requires a running '%s' VM, but there is currently none running.", machineName))
```

Comment:
The execution of this command requires a running '%s' VM, but there is currently none.

no need to repeat running